test/spoom/cli/commands/run_test.rb

```diff
@@ -0,0 +1,125 @@
+# typed: true
+# frozen_string_literal: true
+
+require "pathname"
+
+require_relative "../cli_test_helper"
+
+module Spoom
+  module Cli
+    module Commands
+      class RunTest < Minitest::Test
+        include Spoom::Cli::TestHelper
+        extend Spoom::Cli::TestHelper
+
+        def test_project
+          "project"
+        end
+
+        before_all do
+          install_sorbet("project")
```

Comment:
Do you want to use `test_project` as the argument instead of `"project"`?

Reply:
Thanks, I replaced this by a constant everywhere.